applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -501,7 +501,7 @@ void BaseSolidElement::AddExplicitContribution(
     Vector damping_residual_contribution = ZeroVector(element_size);
 
     // Calculate damping contribution to residual -->
-    if (r_prop.Has(RAYLEIGH_ALPHA) || r_prop.Has(RAYLEIGH_BETA)) {
+    if (r_prop.Has(RAYLEIGH_ALPHA) || r_prop.Has(RAYLEIGH_BETA) || rCurrentProcessInfo.Has(RAYLEIGH_ALPHA) || rCurrentProcessInfo.Has(RAYLEIGH_BETA)) {
```

Comment:
Note, the check for the ProcessInfo is done later, the problem was the initial check

Reply:
You mean that we're already checking for both properties and process info in `CalculateDampingMatrixWithLumpedMass` right?

Reply:
Please use the fct from the element utils, otherwise it is inconsistent with the implicit dynamics

Reply:
Nope, in explicit we can not set the same strategies. The computational cost is increased by a lot and makes the simulation very expensive

Reply:
In fact, I would not consider at all RAYLEIGH_BETA (stiffness damping) as in explicit is totally killer. In fact, the coherent thing would be tom directly provide the damping coefficient in explicit simulations. BTW, someone mentioned that we confuse damping ratio with damping coefficient, we should check that